content/chapters/io/lecture/slides/intro.md

```diff
@@ -0,0 +1,34 @@
+## Backstory
+
+---
+
+### Backstory
+
+- You finish your summer internship
+- You explain the cool internal parts of the project to your friends
+- They hit you with: "How do I interact with it?"
+- You describe the data gathering algorithms again and see them lose interest
+
+What was missing?
+
+----
+
+#### Interactive Software is Good Software
+
+![interactive software](./media/interactive-software.png)
+
+----
+
+#### IO - The Need
```

Comment:
I would replace this with an image showing a process doing computing. And then adding items showing input / output highlighting interactivity.